projects/assets/src/translations/en/product.ts

```diff
@@ -17,8 +17,6 @@ export const product = {
       label: 'Filter by',
       action: 'Filter by',
     },
-    activeFilter:
-      '{{filter}} filter, clicking on this button will remove the filter',
```

Comment:
Is this used in the new asm feature?

Reply:
Don't know why this is here. I didn't make any changes to this file